File: code/AnoNeBot/AnoNeBot/Services/YesNoService.cs
using AnoNeBot.Models;
using Newtonsoft.Json;
using System.Net.Http;
using System.Threading.Tasks;

namespace AnoNeBot.Services
{
    public class YesNoService
    {
        public static async Task<YesNoModel> GetYesNoAsync(bool translate = false)
        {
            HttpResponseMessage res;

            using (HttpClient hc = new HttpClient())
            {
                res = await hc.GetAsync("https://yesno.wtf/api/");
            }

            if (res.IsSuccessStatusCode)
            {
                var yesNo = JsonConvert.DeserializeObject<YesNoModel>(await res.Content.ReadAsStringAsync());
                if (translate)
                    yesNo.Answer = Translate(yesNo.Answer);

                return yesNo;
            }
            else
                return null;
        }

        public static string Translate(string yesNo)
        {
            string translation;

            switch (yesNo.ToLower())
            {
                case Answers.Yes:
                    translation = "Ano";
                    break;
                case Answers.No:
                    translation = "Ne";
                    break;
                default:
                    translation = "Nejsem si jistý";
                    break;
            }

            return translation;
        }
    }
}

File: code/AnoNeBot-Node/services/yes-no-service.js
//@ts-check
const req = require("request-promise")

module.exports.get = (translate = false) => {
    return new Promise((resolve, reject) => {
        req.get("https://yesno.wtf/api/").then((resp) => {
            let r = JSON.parse(resp);
            if (translate)
                r.answer = _translate(r.answer);
            resolve(r);
        }).catch((error) => {
            console.log(`Error calling YesNo API. ${error}`);
            reject(error);
        });
    });
}

function _translate(yesNo) {
    let translation;

    switch (yesNo.toLowerCase()) {
        case "yes":
            translation = "Ano";
            break;
        case "no":
            translation = "Ne";
            break;
        default:
            translation = "Nejsem si jistý";
            break;
    }

    return translation;
}